### PR TITLE
Chore/mp/login

### PR DESCRIPTION
**Pull request by markkopaas**

### CODE REVIEW

File: applications/registration-react/src/app/routes.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import { CatalogsPage } from '../pages/catalogs-page/catalogs-page';
+import { CatalogRoutes } from './catalog-routes';
+
+export const Routes = () => (
+  <Switch>
+    <Route exact path="/loggedOut" render={() => <LoginPage loggedOut />} />
+    <Route exact path="/login" component={LoginPage} />
+    <Route exact path="/">
+      <Redirect to="/catalogs" />
+    </Route>
```
Comment (azaxarov) [inline applications/registration-react/src/app/routes.jsx]: Use rather ```<Redirect exact from="/" to="/catalogs" />```

File: applications/registration-react/src/app/catalog-routes.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <Route
+      exact
+      path="/catalogs/:catalogId/apis/import"
+      component={ConnectedApiImportPage}
+    />
+    <Route
+      exact
+      path="/catalogs/:catalogId/apis/:apiId"
+      component={ApiRegistrationPage}
+    />
```
Comment (azaxarov) [inline applications/registration-react/src/app/catalog-routes.jsx:35]: Maybe add a fallback route? What do we do when users tamper with URL (perhaps by accident)?

**Review by azaxarov — APPROVED** [review]

File: applications/registration-react/src/app/routes.jsx (hunk 1 above)
Comment (markkopaas) [reply applications/registration-react/src/app/routes.jsx]: takk, rettet!

File: applications/registration-react/src/app/catalog-routes.jsx (hunk 2 above)
Comment (markkopaas) [reply applications/registration-react/src/app/catalog-routes.jsx:35]: takk, rettet!

**Pull request merged**